docs_src/src/pages/documentation/example_app/deployment.mdx

```diff
@@ -16,10 +16,10 @@ With the web application deployed and running smoothly, Batman had a powerful ne
 
 <div className="not-prose">
   <Button
-    href="/documentation/example_app/templates"
+    href="/documentation/example_app/redirection"
```

Comment:
Hey @VishnuSanal ,

We should add the route in the `api_reference` folder and not the `example_app` folder